### ISSUE TITLE
binfmt.sh: allow re-running multiple times

### ISSUE THREAD

**Issue by udeved** [OpenRC/openrc#325]
If binfmt.sh is run again after the binfmt service started, it throws an write error that a registered file already exists.
On Artix, like on Arch, we use alpm hooks to re-execute binfmt_misc registration eg on a package update, say mono.

I would propose a simple fix, make sure all binfmt conf are unregistered before registering them again.

I inserted in apply_file() right before writing to register

```
for reg in $(find /proc/sys/fs/binfmt_misc -type f ! -name register ! -name status); do
     echo -1 > $reg
done
```

Any opinions? If this a satisfactory solution, I could do a PR.

**Comment by udeved** [comment]
Answering myself, I think this needs more work, it atm only works if binfmt.sh is not run with a file argument, it would leave all unregistered but only the file arg registered again.

**Comment by vapier** [comment]
this was also reported & fixed at #328
